minefield_game/cell.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+enum class CellStatus : char
+{
+    Empty = 'o',   // Empty cell
+    Guess = '?',   // A guess has been made
+    Bomb  = '*',   // Contains a mine
+    Taken = 'x'    // Disabled cell (can no longer be used)
+};
+
+
+struct Cell
+{
+
+    int x;
+    int y;
+    CellStatus status;
+
+    void setStatus(CellStatus newStatus);
```

Comment:
#should remove - member is publicly accessible and the setter does not do anything other than setting the value